en/04_Changelogs/5.0.0.md

```diff
@@ -62,6 +63,29 @@ This is a major release and as a result there are a number of breaking API chang
     - `SilverStripe\ORM\Connect\Query`
   - Removed `SilverStripe\ORM\DataList::getGenerator()` (use `getIterator()` instead)
   - Removed the `SilverStripe\ORM\Map_Iterator` class. `SilverStripe\ORM\Map` now uses a generator instead.
+- Removed various API related to `SwiftMailer`
+  - Removed the following classes and interfaces
+    - `SilverStripe\Control\Email\Mailer`
+    - `SwiftMailer`
+    - `SwiftPlugin`
+  - Removed the following methods from `SilverStripe\Control\Email\Email`
+    - Replaced with `Email::getData()`
+      - `BaseURL()`
+      - `IsEmail()`
+    - Has equivalent functionality via `Email::getTextBody()`
```

Comment:
Why is the above "replaced with" but this is "has equivalent functionality via"?
IMO they should all say "replaced with" as that's much clearer language.

Reply:
Updated